cmd/kube-scheduler/app/server.go

```diff
@@ -169,6 +165,10 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}, regis
 		}
 	}
 
+	// Apply algorithms based on feature gates.
+	// TODO: make configurable?
+	algorithmprovider.ApplyFeatureGates()
```

Comment:
Right now we don't need to move this call, right?

Reply:
Got feedback from @ravisantoshgudimetla to move it here https://github.com/kubernetes/kubernetes/pull/79426#discussion_r298661507